### PR TITLE
docs: change PascalCase parameters to snake_case style

### PR DESCRIPTION
**Pull request by classicalliu**

### CODE REVIEW

File: rpc/doc.md [hunk 1]
```diff
@@ -182,9 +181,9 @@ Returns the information about cells collection by type_hash.
 
 ## Parameters
 
-    Type_hash - Cell type_hash.
-    From - Start BlockNumber.
-    To - End BlockNumber.
+    type_hash - Cell type_hash.
```
Comment (doitian) [inline rpc/doc.md]:
```suggestion
    type_hash - Cell type hash.
```

The part after `-` is a description, I don't think we need the underscore in it. So does the following two lines.

File: rpc/doc.md [hunk 2]
```diff
@@ -182,9 +181,9 @@ Returns the information about cells collection by type_hash.
...
+    from - Start block_number.
```
Comment (doitian) [inline rpc/doc.md]:
```suggestion
    from - Start block number.
```

File: rpc/doc.md [hunk 3]
```diff
@@ -182,9 +181,9 @@ Returns the information about cells collection by type_hash.
...
+    to - End block_number.
```
Comment (doitian) [inline rpc/doc.md]:
```suggestion
    to - End block number.
```

File: rpc/doc.md (hunk 1 above)
Comment (classicalliu) [reply rpc/doc.md]: Nice, I'll update it.

**Pull request merged**